model-server/build.gradle.kts

```diff
@@ -184,3 +187,77 @@ spotless {
                 '\n'*/
     }
 }
+
+// OpenAPI integration
+val basePackage = project.group.toString()
+val openAPIgenerationPath = "$buildDir/generated/openapi"
+
+// Pairs of the different OpenAPI files we use. Each pair must have its own 'category' as first argument as these
+// are used to generate corresponding packages
+val openApiFiles = listOf(
+    Pair("public", "model-server"),
```

Comment:
just FYI: You could have used `"public" to "model-server"` to create a `Pair`.

Reply:
✅